Fits/Electronic Attack Frigates/[ADV-30] Keres.md

```diff
@@ -0,0 +1,46 @@
+# Keres
+
+Advanced EWAR Frigate
+
+
+## Description
+
+Utilizes Remote Sensor Dampeners (Damps) to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+Targeting Range - Use on long range platforms like enemy ECM or ships with long engagement ranges.
+Scan Resolution - Use on Logistics ships that are close to the enemy fleet or against any ship that is too close for targeting range to make a difference.
+
+Small gang refit should replace the meta Dampener for the T2 Warp Disruptor and remove the cloak.
+
+## Fit
+
+```
+[Keres, [ADV-30] Keres]
+
+IFFA Compact Damage Control
+Imperial Navy 200mm Steel Plates
+Small Ancillary Armor Repairer
+
+5MN Cold-Gas Enduring Microwarpdrive
+Remote Sensor Dampener II
+Remote Sensor Dampener II
+Remote Sensor Dampener II
+Phased Muon Scoped Sensor Dampener
+
+Prototype Cloaking Device I
+[Empty High slot]
```

Comment:
Auto targeting might be good, gives more targets

Reply:
They get 9 targets by default so it felt overkill lol